### PR TITLE
Fixes #10269: Doc about copying ncf technique to /var/rudder/ncf/local is false

### PR DESCRIPTION
**Pull request by fanf**
https://www.rudder-project.org/redmine/issues/10269

### CODE REVIEW

File: 4_advanced_usage/30_technique_creation.txt [hunk 1]
```diff
@@ -5,95 +5,112 @@ configuration and system administration needs. You can also create your own
...
 
-* /var/rudder/configuration-repository/ncf
+* `/var/rudder/configuration-repository/ncf`
 ** This is were you add your own ncf Generic Methods and Techniques.
-Techniques created with the Technique Editor will be located here, and both Generic Methods
-and Techniques in that place will be accessible in the Technique Editor alongside what is 
-provided by the standard library. 
+   Techniques created with the Technique Editor will be located here, and both
+   Generic and Techniques in that place will be accessible in the Technique Editor
+   alongside what provided by the standard library.
```
Comment (amousset) [inline 4_advanced_usage/30_technique_creation.txt]: what is

File: 4_advanced_usage/30_technique_creation.txt [hunk 2]
```diff
@@ -5,95 +5,112 @@ configuration and system administration needs. You can also create your own
...
 
-To share those folders to all nodes, Rudder makes a copy of these folders in /var/rudder/ncf:
+==== Sharing ncf code with nodes
 
-* /var/rudder/ncf/local is a copy of /var/rudder/configuration-repository/ncf
-* /var/rudder/ncf/common is a copy /usr/share/ncf/tree
+To share those folders to all nodes, Rudder makes a copy of these folders in two
+place:
```
Comment (amousset) [inline 4_advanced_usage/30_technique_creation.txt]: places

File: 4_advanced_usage/30_technique_creation.txt [hunk 3]
```diff
@@ -5,95 +5,112 @@ configuration and system administration needs. You can also create your own
...
 
-A node updates its ncf local library by copying the content of these two folders during its
-promise update phase. 
 
-===== Technique Editor integration with Rudder 
 
-Here we will explain Technique Editor behavior and what workflow are initialized by different 
-action on the Technique Editor.
+Files in `/var/rudder/ncf` are synchronized automatically by the "rudder agent update"
+command execute when the agent run on the server. So any modification done in files
```
Comment (amousset) [inline 4_advanced_usage/30_technique_creation.txt]: s/execute//

Comment (amousset) [inline 4_advanced_usage/30_technique_creation.txt]: the agent runs

File: 4_advanced_usage/30_technique_creation.txt [hunk 4]
```diff
@@ -315,4 +332,3 @@ Cf http://www.rudder-project.org/foswiki/Development/Technique
 *WORK IN PROGRESS*
 The reports format
 Cf http://www.rudder-project.org/foswiki/Development/ReportsInTechniques
-
```
Comment (amousset) [inline 4_advanced_usage/30_technique_creation.txt]: We need to keep the newline here

**Comment by fanf** [comment]
PR rebased

**Pull request merged**